### ISSUE TITLE
NgbTypeahead missing required attribute aria-controls

### ISSUE THREAD

**Issue by ruszabarov** [ng-bootstrap/ng-bootstrap#4815]
NgbTypeahead has `role=combobox`, which requires `aria-controls` attribute to be set to the `id` of the popup `listbox`, `tree`, `grid`, or `dialog` element. Currently this fails Axe accessibility check.

**Comment by ruszabarov** [comment]
created PR https://github.com/ng-bootstrap/ng-bootstrap/pull/4816
